crates/qt-gui-qquickpainteditem/Cargo.toml

```diff
@@ -0,0 +1,15 @@
+[package]
+name = "qt-gui"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+rustagram2="2"
```

Comment:
could have spaces around `=` and on line below and in the git ones below

Reply:
Done